qa/os/src/test/java/org/elasticsearch/packaging/util/Archives.java

```diff
@@ -275,26 +259,45 @@ public static void runElasticsearch(Installation installation, Shell sh) throws
 
         Platforms.onWindows(() -> {
             // this starts the server in the background. the -d flag is unsupported on windows
-            // these tests run as Administrator. we don't want to run the server as Administrator, so we provide the current user's
-            // username and password to the process which has the effect of starting it not as Administrator.
-            sh.run(
-                "$password = ConvertTo-SecureString 'vagrant' -AsPlainText -Force; " +
-                "$processInfo = New-Object System.Diagnostics.ProcessStartInfo; " +
-                "$processInfo.FileName = '" + bin.elasticsearch + "'; " +
-                "$processInfo.Arguments = '-p " + installation.home.resolve("elasticsearch.pid") + "'; " +
-                "$processInfo.Username = 'vagrant'; " +
-                "$processInfo.Password = $password; " +
-                "$processInfo.RedirectStandardOutput = $true; " +
-                "$processInfo.RedirectStandardError = $true; " +
-                sh.env.entrySet().stream()
-                    .map(entry -> "$processInfo.Environment.Add('" + entry.getKey() + "', '" + entry.getValue() + "'); ")
-                    .collect(joining()) +
-                "$processInfo.UseShellExecute = $false; " +
-                "$process = New-Object System.Diagnostics.Process; " +
-                "$process.StartInfo = $processInfo; " +
-                "$process.Start() | Out-Null; " +
-                "$process.Id;"
-            );
+            if (System.getenv("username").equals("vagrant")) {
```

Comment:
Most of the PowerShell lines in the following `sh.run()` calls are the same. Is there any value in reducing the duplication here?